policyengine_us/variables/gov/states/wv/tax/income/subtractions/public_pension/wv_public_pension_subtraction_person.py

```diff
@@ -1,28 +1,24 @@
 from policyengine_us.model_api import *
 
 
-class wv_public_pension_subtraction(Variable):
+class wv_public_pension_subtraction_person(Variable):
     value_type = float
-    entity = TaxUnit
-    label = "West Virginia public pension subtraction"
+    entity = Person
+    label = "West Virginia public pension subtraction for each person"
     defined_for = StateCode.WV
     unit = USD
     definition_period = YEAR
     reference = "https://code.wvlegislature.gov/11-21-12/"
 
-    def formula(tax_unit, period, parameters):
-        person = tax_unit.members
+    def formula(person, period, parameters):
         # "benefits received under any federal retirement system to which Title 4 U.S.C. §111 applies"
         # https://www.law.cornell.edu/uscode/text/4/111
         federal_pension_income = person(
             "taxable_federal_pension_income", period
         )
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        head_or_spouse_pension_income = federal_pension_income * head_or_spouse
         p = parameters(
             period
         ).gov.states.wv.tax.income.subtractions.public_pension
-        person_capped = min_(federal_pension_income, p.max_amount)
-        # Only applies to head and spouse.
-        head_spouse_capped = person_capped * ~person(
-            "is_tax_unit_dependent", period
-        )
-        return tax_unit.sum(head_spouse_capped)
+        return min_(head_or_spouse_pension_income, p.max_amount)
```

Comment:
Do this instead of `head_or_spouse_pension_income` to more clearly indicate eligibility criteria
```suggestion
        capped = min_(federal_pension_income, p.max_amount)
        return capped * head_or_spouse
```